heudiconv/dicoms.py

```diff
@@ -314,19 +315,55 @@ def group_dicoms_into_seqinfos(files, grouping, file_filter=None,
     return seqinfos
 
 
-def get_dicom_series_time(dicom_list):
-    """Get time in seconds since epoch from dicom series date and time
-    Primarily to be used for reproducible time stamping
+def get_timestamp_from_series(dicom_list: list[str]) -> int:
+    """try to return a timestamp indicating when the first dicom in dicom_list was created, or if that
+    info isn't present then a reproducible integer. This is used in setting mtimes reproducibly
```

Comment:
```suggestion
    """Get time in seconds since epoch from DICOM series date and time
    
    Returns a timestamp indicating when the first DICOM in ``dicom_list`` was created.
    If that info isn't present then a reproducible integer based on ``SeriesInstanceUID``.
    This is used in setting mtimes reproducibly.
```